public/test/spec/autocomplete.spec.js

```diff
@@ -42,7 +47,12 @@ describe('Autocomplete', function () {
             autocomplete({
                 query: 'banana',
                 path: 'fruit'
-            }).then(done.fail, done);
+            }).then(() => {
```

Comment:
I'd be really interested to know what changed for this to be necessary! 👍

Reply:
It was the version of karma which changed this. But this is better anyway because we are actually checking the error string here!